amazon-s3/6.0/modules/ROOT/pages/amazon-s3-connector-examples.adoc

```diff
@@ -13,7 +13,7 @@ For this example to work in Anypoint Studio, you must provide Amazon Web Service
 [[store-and-retrieve]]
 == Example Steps
 
-image::amazon-s3-02.png["Studio 7 Visual Studio Icon Flow. The flow shows Listener, Create bucket, Get MuleSoft logo, Create logo object in S3 bucket, Get Image, and Delete S3 bucket."]
+image::flow.png["Studio 7 Visual Studio Icon Flow. The flow shows Listener, Create bucket, Get MuleSoft logo, Create logo object in S3 bucket, Get Image, Delete object and Delete S3 bucket."]
```

Comment:
Is there a source called Listener? If not, use the whole source name. If you are referring to a generic listener, then use lower case.